circom/tests/controlflow/early_return_loop_with_unknown_if.circom

```diff
@@ -2,6 +2,7 @@ pragma circom 2.0.3;
 // REQUIRES: circom
 // RUN: rm -rf %t && mkdir %t && %circom --llvm -o %t %s | sed -n 's/.*Written successfully:.* \(.*\)/\1/p' | xargs cat | FileCheck %s --enable-var-scope
 
+// %0 (i.e. arena) = [ a[0], a[1], b[0], b[1], i ]
```

Comment:
I'm a fan of these style of comments, makes the examples easier to parse for what we're looking for in the lit tests.